src/main/java/uk/gov/digital/ho/hocs/workflow/api/WorkflowConstants.java

```diff
@@ -29,5 +29,6 @@ public interface WorkflowConstants {
     String CHANNEL = "Channel";
     String CHANNEL_COMP_ORIGINATEDFROM = "XOriginatedFrom";
     String CHANNEL_COMP_WEBFORM = "Webform";
-
+    String PREVIOUS_VALUE_KEY = "previous%s";
+    String DATA_FIELD_UPDATED_EVENT_NAME = "%s_UPDATED";
```

Comment:
```suggestion
    String DATA_FIELD_UPDATED_EVENT_NAME = "CASE_DATA_ITEM_UPDATED";
```

Would this not just be this - we would have enough in the FE then to string interpolation for the UI.